mercury_app/tests.py

```diff
@@ -812,6 +813,18 @@ def test_get_context_data(self):
         self.assertIsNotNone(response.context['event'])
         self.assertIsNotNone(response.context['data_tipes_handed'])
 
+    @patch('mercury_app.views.get_auth_token', return_value=123123)
+    @patch('mercury_app.views.get_api_orders_of_event')
+    def test_get_context_data_create_orders(self, mock_api_orders, mock_get_token):
+        event = EventFactory()
+        self.assertEqual(Order.objects.filter(event=event).count(), 0)
+        mock_api_orders.return_value = get_mock_api_orders(
+            1, 1, event.eb_event_id)
+        response = self.client.get(
+            '/event/{}/summary/'.format(event.eb_event_id),
+        )
+        self.assertIsNotNone(Order.objects.filter(event=event).count(), 1)
```

Comment:
Are you checking is not None o equal to 1?

Reply:
Equal to one, I was wrong! Thanks!